fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/MetaDataProtoEditorUnitTest.java

```diff
@@ -100,17 +130,17 @@ public void fieldIsTypeUnqualified() throws Descriptors.DescriptorValidationExce
         assertNotNull(simpleRecordDescriptor);
         assertSame(simpleRecordDescriptor, modifiedFileDescriptor.findMessageTypeByName(RecordMetaDataBuilder.DEFAULT_UNION_NAME).findFieldByName("_MySimpleRecord").getMessageType());
 
-        assertEquals(FieldTypeMatch.MIGHT_MATCH,
+        assertEquals(FieldTypeMatch.MATCHES,
```

Comment:
I'm sure you felt this while you were trying to edit these tests, but this does seem like the kind of thing that we'd want to consider refactoring into a parameterized test. It probably would have sped up fixing all of these